packages/fiori/src/DynamicPageTitle.ts

```diff
@@ -132,6 +135,22 @@ class DynamicPageTitle extends UI5Element {
 	@slot({ type: HTMLElement })
 	snappedHeading!: HTMLElement[];
 
+	/**
+	 * Defines the content of the snapped title on mobile devices.
+	 *
+	 * This slot is displayed only when the `DynamicPageTitle` is in the snapped state on mobile devices.
+	 * It should be used to provide a simplified, single-line title that takes up less space on smaller screens.
+	 *
+	 * **Note:**
+	 * - The content set in this slot **overrides** all other content set in the `DynamicPageTitle` slots when displayed.
+	 * - The slot is intended for a single `ui5-title` component.
+	 *
+	 * @public
+	 * @since 2.3.0
+	 */
+	@slot({ type: Title })
+	snappedTitleOnMobile!: Array<Title>;
```

Comment:
I am okay leaving it Array<Title>, but in some places we intentionally give freedom to be HTMLElement BUT we recommend given component (title in this case).

But as I said, we can leave it as it is.

Reply:
Thanks for the comment! I decided to approach it this way because I believe it’s a good idea to limit the set of controls that can be placed in that slot. I was guided by the [example](https://openui5nightly.hana.ondemand.com/api/sap.f.DynamicPageTitle#aggregations) we have in OpenUI5, where the `snappedTitleOnMobile` aggregation accepts a single control of type `sap.m.Title`.